contracts/token/ERC721/ERC721Receiver.sol

```diff
@@ -21,12 +21,14 @@ contract ERC721Receiver {
    * transfer. This function MUST use 50,000 gas or less. Return of other
    * than the magic value MUST result in the transaction being reverted.
    * Note: the contract address is always the message sender.
-   * @param _from The sending address
+   * @param _operator The address which called `safeTransferFrom` function
+   * @param _from The address which previously owned the token
    * @param _tokenId The NFT identifier which is being transfered
    * @param _data Additional data with no specified format
    * @return `bytes4(keccak256("onERC721Received(address,uint256,bytes)"))`
```

Comment:
please update the text here as well

Reply:
Done. Also fixed the "This function MUST use 50,000 gas or less." statement, which is no longer true.